### PR TITLE
codestyle

### PR DESCRIPTION
**Pull request by ximex**
unify indentation in config.default.php
unify javascript codestyle in sticky-headers.js

### CODE REVIEW

File: config/config.default.php [hunk 1]
```diff
@@ -278,7 +278,7 @@
     'enable_goody'            => (bool)env('ENABLE_GOODY', false),
 
     // Enables the food voucher in the user profile
-    'enable_voucher'            => (bool)env('ENABLE_VOUCHER', true),
+    'enable_voucher'          => (bool)env('ENABLE_VOUCHER', true),
```
Comment (weeman1337) [inline config/config.default.php:281]: Can you try to add a rule that covers this style to the [phpcs rule set](https://github.com/engelsystem/engelsystem/blob/main/.phpcs.xml)?

File: resources/assets/js/sticky-headers.js [hunk 2]
```diff
@@ -14,19 +14,18 @@ $(function () {
...
-        timeLanes.css({
-          'left': Math.max(0, $(window).scrollLeft() - left) + 'px'
-        });
-        headers.css({
-          'top': Math.max(0, $(window).scrollTop() - top
-                        - 13
-                        + topReference.offset().top)
-                        + 'px'
-        });
+    $(window).scroll(function () {
```
Comment (weeman1337) [inline resources/assets/js/sticky-headers.js:17]: Can you try to add a rule that covers this style to the [ESLint rule set](https://github.com/engelsystem/engelsystem/blob/main/.eslintrc.json)?

**Review by weeman1337 — COMMENTED** [review]
I would say we should not format code if it passes the code style linters. Otherwise people will format code back and forth.

File: resources/assets/js/sticky-headers.js (hunk 2 above)
Comment (ximex) [reply resources/assets/js/sticky-headers.js:17]: done

File: config/config.default.php (hunk 1 above)
Comment (ximex) [reply config/config.default.php:281]: never worked with phpcs. i only aligned it as all the other indentations

Comment (weeman1337) [reply config/config.default.php:281]: Never mind. I think this only works for direct siblings.

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Can you rebase your PR and re-request review?

**Review by weeman1337 — APPROVED** [review]

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Can you fix the current ESLint errors and then re-request review?

```
~/git/chaos/engelsystem (codestyle)$ yarn run lint
yarn run v1.22.18
$ eslint .

/home/weeman/git/chaos/engelsystem/resources/assets/js/dashboard.js
  28:32  error  Unexpected newline after '('   function-paren-newline
  30:7   error  Unexpected newline before ')'  function-paren-newline

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Review by weeman1337 — APPROVED** [review]
Looks good now. Thanks :+1:

**Pull request merged**